sdk/src/client/stronghold/error.rs

```diff
@@ -51,6 +51,9 @@ pub enum Error {
     /// Path already exists
     #[error("path already exists: {0}")]
     PathAlreadyExists(std::path::PathBuf),
+    /// Path is not a file
+    #[error("path is not a file: {0}")]
+    PathIsNotFile(std::path::PathBuf),
```

Comment:
Can't do that, breaking change

Reply:
i argue its not. the code wouldt allow it anyway, because the filesystem would lock and crash. This just gives a better error message. And there is absolutely NO way someone has a match case for all errors :)

Reply:
We had such a case 2 weeks ago and we put TODOs to change later in 2.0 and found another way. This is breaking change whether people use it that way or not.

Reply:
So what error should i throw? IO?